llvm/include/llvm/CodeGen/AccelTable.h

```diff
@@ -257,17 +257,37 @@ class AppleAccelTableData : public AccelTableData {
 
 /// Helper class to identify an entry in DWARF5AccelTable based on their DIE
 /// offset and UnitID.
-struct OffsetAndUnitID : std::pair<uint64_t, uint32_t> {
```

Comment:
would it make it sense to just use tuple here to avoid having to define the dense map boilerplate?

Reply:
I don't really have a strong opinion. Poking around I don't really see tuple being used with DenseMap. More standard approach seems this boilerplate code. 
@dwblaikie to break a tie? :)